### PR TITLE
MC-36350: Update documentation about Adjustment Refund option on Cred…

### PR DESCRIPTION
**Pull request by thiaramus**
Update documentation about Adjustment Refund option on Credit Memo page

## Purpose of this pull request

After MC-33829 was resolved we need to update documentation about the Adjustment Refund option on the Credit Memo page. Product Owner approval for this change could be found here MC-36350.

## Magento release version

- [x] 2.4.x
- [x] 2.3.6

## Product editions

Is this update specific to a product edition?

- [x] Magento Open Source only
- [x] Magento Commerce only

## Is this update specific to an installed feature extension

- [ ] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

- https://docs.magento.com/user-guide/sales/credit-memo-create.html

## Links to Magento source code or PRs

- https://github.com/magento/magento2ee/pull/2372

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by leslietilling** [comment]
@thiaramus  Looks like you are a Magento employee. Can you slack me your name so we can include you in the internal contributors list?

File: src/sales/credit-memo-create.md [hunk 1]
```diff
@@ -74,7 +74,7 @@ _Create Credit Memo_
 
       This field initially shows the total shipping amount from the order that is available for refund. It is equal to the full shipping amount from the order, less any shipping amount that has already been refunded. Like the quantity, the amount can be reduced, but not increased.
 
-   - For **Adjustment Refund**, enter a value to be added to the total amount refunded as an additional refund that does not apply to any particular part of the order (shipping, items, or tax).
+   - For **Adjustment Refund**, enter a value to be added to the total amount refunded as an additional refund that does not apply to any particular part of the order (shipping, items, or tax). It also can be used for partial refund with virtual money like Gift Card when an administrator wants to refund a non-virtual payment method first.
```
Comment (leslietilling) [inline src/sales/credit-memo-create.md:77]:
```suggestion
   - For **Adjustment Refund**, enter a value to be added to the total amount refunded as an additional refund that does not apply to any particular part of the order (shipping, items, or tax). It also can be used for partial refund with virtual money, such as a gift card, when an administrator wants to refund a non-virtual payment method first.
```

File: src/sales/credit-memo-create.md [hunk 2]
```diff
@@ -135,7 +135,7 @@ _Create Credit Memo_
 
       This field initially displays the total shipping amount from the order that is available for refund. It is equal to the full shipping amount from the order, less any shipping amount that has already been refunded. Like the quantity, the amount can be reduced, but not increased.
 
-   - For **Adjustment Refund**, enter a value to be added to the total amount refunded as an additional refund that does not apply to any particular part of the order (shipping, items, or tax).
+   - For **Adjustment Refund**, enter a value to be added to the total amount refunded as an additional refund that does not apply to any particular part of the order (shipping, items, or tax). It also can be used for partial refund with virtual money like Gift Card when an administrator wants to refund a non-virtual payment method first.
```
Comment (leslietilling) [inline src/sales/credit-memo-create.md:138]:
```suggestion
   - For **Adjustment Refund**, enter a value to be added to the total amount refunded as an additional refund that does not apply to any particular part of the order (shipping, items, or tax). It also can be used for partial refund with virtual money, such as a gift card, when an administrator wants to refund a non-virtual payment method first.
```

**Comment by leslietilling** [comment]
running tests

**Pull request merged**